tests/test_models.py

```diff
@@ -19,6 +22,10 @@
 from figo.models import Transaction
 from figo.models import User
 
+from tests.test_writing_methods import CLIENT_ERROR
+
+HTTP_NOT_ACCEPTABLE = 406
```

Comment:
Wouldn't it be better to have something like a constants module also for the SDK?

Reply:
`httplib.NOT_ACCEPTABLE`